rpcs3/Emu/Cell/Modules/cellOskDialog.cpp

```diff
@@ -20,13 +20,16 @@ s32 cellOskDialogLoadAsync(u32 container, vm::ptr<CellOskDialogParam> dialogPara
 
 	if (!inputFieldInfo || !inputFieldInfo->message || !inputFieldInfo->init_text || inputFieldInfo->limit_length > CELL_OSKDIALOG_STRING_SIZE)
 	{
+		cellOskDialog.error("cellOskDialogLoadAsync: CELL_OSKDIALOG_ERROR_PARAM");
```

Comment:
It's like `error_code` never existed.

Reply:
It wouldn't compile

Reply:
You were probably missing required string formatting for the error codes.